cmd/osbuild-worker/jobimpl-osbuild.go

```diff
@@ -62,6 +62,8 @@ func validateResult(result *worker.OSBuildJobResult, jobID string) {
 		logWithId.Errorf("osbuild job failed: %s", reason)
 		result.JobError = clienterrors.WorkerClientError(clienterrors.ErrorBuildJob, reason)
 		return
+	} else {
+		logWithId.Infof("koji-build job succeeded")
```

Comment:
s/koji-build/osbuild/

Reply:
Right, I guess I had hope we could stick to a single log message to make filtering for it simpler :smiley: